### PR TITLE
Rename helmfile.d to releases

### PR DESCRIPTION
**Pull request by goruha**
## What
* Rename `helmfile.d` to `releases`
* Update documentation instructions for installation

## Why
* To solve `helmfile` conflict between `helmfile.yaml` and `helmfile.d`

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ usage: |-
   ```
   FROM cloudposse/helmfiles:latest as helmfiles
 
-  COPY --from helmfiles /helmfile.d/ /conf/helmfiles.d/
+  COPY --from helmfiles /releases/ /conf/releases/
```
Comment (osterman) [inline README.yaml]: What do think about using the `/conf/helmfiles/releases` pattern?

Comment (osterman) [reply README.yaml]: so there would be a `/conf/helmfiles/scripts` and `/conf/helmfiles/helmfile.yaml`

**Comment by osterman** [comment]
@goruha rebase

File: README.yaml (hunk 1 above)
Comment (goruha) [reply README.yaml]: Agree

File: README.md [hunk 2]
```diff
@@ -67,7 +67,8 @@ We distribute a docker image with all helmfiles. Use a multi-stage build to incl
 ```
 FROM cloudposse/helmfiles:latest as helmfiles
 
-COPY --from helmfiles /helmfile.d/ /conf/helmfiles.d/
+COPY --from helmfiles /releases/ /conf/helmfiles/releases/
```
Comment (osterman) [inline README.md]:
```suggestion
COPY --from helmfiles /helmfiles/ /conf/helmfiles/
```

File: README.yaml [hunk 3]
```diff
@@ -47,7 +47,8 @@ usage: |-
   ```
   FROM cloudposse/helmfiles:latest as helmfiles
 
-  COPY --from helmfiles /helmfile.d/ /conf/helmfiles.d/
+  COPY --from helmfiles /releases/ /conf/helmfiles/releases/
```
Comment (osterman) [inline README.yaml]:
```suggestion
  COPY --from helmfiles /helmfiles/ /conf/helmfiles/
```

File: README.yaml [hunk 4]
```diff
@@ -47,7 +47,8 @@ usage: |-
...
+  COPY --from helmfiles /scripts/ /conf/helmfiles/scripts/
```
Comment (osterman) [inline README.yaml]:
```suggestion
```

**Review by osterman — CHANGES_REQUESTED** [review]
See comments

File: README.yaml (hunk 4 above)
Comment (goruha) [reply README.yaml]: Scripts required for `kiam`

Comment (goruha) [reply README.yaml]: @osterman ^

**Review by osterman — APPROVED** [review]

**Pull request merged**